app/src/main/java/com/celzero/bravedns/ui/DNSConfigureWebViewActivity.kt

```diff
@@ -232,7 +232,7 @@ class DNSConfigureWebViewActivity : AppCompatActivity(R.layout.activity_faq_webv
                 }
             }
         } else {
-            persistentState.setLocalBlockListStamp(stamp)
+            persistentState.localBlockListStamp = stamp
```

Comment:
again, what's the point of setting these in both `if` and in `else`? simplify the code, pls.

Reply:
Moved the code above `if-else` block